Form/FormFlowInterface.php

```diff
@@ -27,9 +27,9 @@ function getName();
 	function setFormFactory(FormFactoryInterface $formFactory);
 
 	/**
-	 * @param Request|null $request
+	 * @param RequestStack $requestStack
 	 */
-	function setRequest(Request $request = null);
+	function setRequestStack(RequestStack $requestStack);
```

Comment:
It's personal preference and simplifies testing. At least it doesn't need to be changed for this PR.